src/ripple/app/misc/ValidatorList.h

```diff
@@ -125,6 +125,14 @@ class ValidatorList
 
     PublicKey localPubKey_;
 
+    // This is the minimum quantity of validators from which a resizeable
+    // trusted set can be created. In other words, if the size of the list of
+    // all validators is less, then use all of them in the trusted set.
+    int const MINIMUM_RESIZEABLE_UNL {24};
```

Comment:
I'll echo @nbougalis's that the comment here is confusing. Perhaps something like

> The minimum number of listed validators required to allow removing non-communicative validators from the trusted set.   In other words, if the number of listed validators is less, then use all of them in the trusted set.